qiita_ware/ebi.py

```diff
@@ -674,25 +644,7 @@ def send_xml(self):
                 study_accession = accessions.group('study')
                 submission_accession = accessions.group('submission')
 
-                LogEntry.create('Runtime', "Study accession:\t%s" %
-                                study_accession)
-                LogEntry.create('Runtime', "Submission accession:\t%s" %
-                                submission_accession)
-
-                print "Study accession:\t", study_accession
-                print "Submission accession:\t", submission_accession
-            else:
-                LogEntry.create('Runtime', ("However, the accession numbers "
-                                            "could not be found in the output "
-                                            "above."))
-                print ("However, the accession numbers could not be found in "
-                       "the output above.")
-        else:
-            LogEntry.create('Fatal', curl_result)
-            print curl_result
-            print "FAILED"
-
-        return (study_accession, submission_accession)
+        return study_accession, submission_accession
```

Comment:
I think this function should be the one that raises the error if the ebi submission failes (i.e. `success="false"`).
This function has the result so we can actually do some intelligent parsing to raise different type of errors. I know that right now we might not now those different types of errors, but one think that we can do here is raise an error an include the curl_result in the error, so we know what went wrong and we can debug way easier. That error should be both in a LogEntry and in a raised exception (either a new type for EBI or some of the existing ones).